### PR TITLE
Relationship labels

### PR DESCRIPTION
**Pull request by dangoslen**
Implements #48, though, without supporting the flag in the command line. That seemed like it would be too tedious for a user anyway

I'm considering using a YAML map instead of an array, though it would require using a complex key. I'm open to whichever you prefer

I also plan to refactor the `RelationshipLabels` to become a map for better runtime lookup.

### CODE REVIEW

File: .go-version
Comment (dangoslen) [inline .go-version:1]: This can be helpful for users of [goenv](https://github.com/go-nv/goenv/tree/master)

File: config/relationship_label.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	}
+
+	return RelationshipLabel{
+		PkName: string(groups[1]),
+		FkName: string(groups[2]),
+		Label:  string(groups[3]),
+	}, nil
+}
+
+var labelRegex = regexp.MustCompile(`([\w\._-]+)[\s]+([\w\._-]+)[\s]+:[\s]+([\w._-]+)`)
```
Comment (dangoslen) [inline config/relationship_label.go:44]: Will add comments here for the details of the regex

File: diagram/diagram.go [hunk 2]
```diff
@@ -59,12 +59,13 @@ func (d diagram) Create(result *database.Result) error {
 	}
 
 	var constraints []ErdConstraintData
+	relationshipLabelMap := BuildRelationshipLabelMap(d.config)
```
Comment (dangoslen) [inline diagram/diagram.go:62]: Building the map is `O(n)` and we then get (essentially) `O(1)` lookup times when finding a constraint later vs doing a `O(n)` linear search each time

**Review by KarnerTh — APPROVED** [review]
Nice work - thanks for the contribution 👍🏼

**Pull request merged**